src/smspark/job.py

```diff
@@ -32,6 +32,9 @@
 class ProcessingJobManager(object):
     """Manages the lifecycle of a Spark job."""
 
+    _bootstrapping_timeout = 600.0  # all hosts should report as ready within this timeout.
+    _wait_for_primary_timeout = 600.0  # then, all workers ask the primary if it's up within this timeout.
+
```

Comment:
Could you track a follow-up to merge this into the `master` branch too? Just to make sure they don't end up getting out of sync.

Reply:
sure, eventually all branches will be merged into mainline, including this one.